webview/src/experiments/components/table/MergeHeaderGroups.tsx

```diff
@@ -28,16 +31,21 @@ export const MergedHeaderGroups: React.FC<{
   orderedColumns,
   onDragUpdate,
   onDragEnd,
-  onDragStart
+  onDragStart,
+  root,
+  isFirst,
+  setExpColumnNeedsShadow
 }) => {
   return (
     <div
       {...headerGroup.getHeaderGroupProps({
         className: cx(styles.tr, styles.headRow)
       })}
     >
-      {headerGroup.headers.map((column: HeaderGroup<Experiment>) => (
+      {headerGroup.headers.map((column: HeaderGroup<Experiment>, ind) => (
         <TableHeader
+          isFirst={isFirst && ind === 0}
```

Comment:
[Q] Why do we need both of these criteria? 
[Q] Can we pull out the `index === 0` into a function so we don't duplicate it in two different places?

Reply:
> [Q] Why do we need both of these criteria?

We need to select just one experiment column cell (aka one of the rendered `TableHeader`s) so we can add the observer to it. The PR does this by finding the first `MergedHeaderGroups`, then the first `TableHeader`. Personally, I felt like this solution felt overcomplicated but I couldn't think of an alternative 🤔 

> [Q] Can we pull out the index === 0 into a function so we don't duplicate it in two different places?

Sure!